src/test/resources/org/rust/lang/core/parser/fixtures/partial/exprs.rs

```diff
@@ -9,4 +9,6 @@ fn foo() {
     // Distinguish `for` expr from `for<>` in types
     let _:
     for a in b {}
+
+    let _ = 0.0.0e92;
```

Comment:
Hm, why we parse this as a float literal?
The compiler produces `error[E0610]: `{float}` is a primitive type and therefore doesn't have fields` here, parser in master shows some parser error, parser with these changes doesn't show any error

Reply:
Fixed